.github/workflows/go.yml

```diff
@@ -0,0 +1,49 @@
+name: Lint and Test
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Run golangci-lint 1.23.8
+        uses: docker://reviewdog/action-golangci-lint:v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          golangci_lint_flags: "--config=.golangci.yml"
+          level: error
+          reporter: github-pr-review
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
```

Comment:
Is it really necessary to checkout the code on every action "target" (not sure what the official term is).

Reply:
I believe so, jobs run in parallel in separate environments so each one will need to checkout to code independently

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs

Reply:
Thanks.